src/api/app/controllers/status/reports_controller.rb

```diff
@@ -1,41 +1,22 @@
 class Status::ReportsController < ApplicationController
-  before_action :set_status_report, only: [:show, :destroy]
   before_action :require_checkable
-  skip_before_action :require_login, only: [:show, :index]
-  after_action :verify_authorized, except: [:show, :index]
+  before_action :set_status_report
+  skip_before_action :require_login
```

Comment:
Please limit this to `:show`, otherwise the next developer will add another action that will then be accessible to everybody without him knowing...